### PR TITLE
feat(ux): flatten `wash ctl stop` into `wash stop`

### PR DESCRIPTION
**Pull request by vados-cosmonic**
## Feature or Problem

Flatten `wash ctl stop` into `wash stop`

See:
- https://github.com/wasmCloud/wash/issues/538

## Release Information

`next`

## Consumer Impact

Consumers will be able to use a more streamlined interface to wash commands

## Testing

Built on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

### Acceptance or Integration

Add integration tests for `wash stop actor`, `wash stop host`, `wash stop provider`

### Manual Verification

Verified locally

### CODE REVIEW

**Comment by vados-cosmonic** [comment]
a pre-existing test `integration_up_can_start_wasmcloud_and_actor` test is failing due to the change in command output  (now printing the generated actor ID), fixing...

File: crates/wash-lib/src/cli/start.rs [hunk 1]
```diff
@@ -102,12 +118,17 @@ pub async fn start_actor(cmd: StartActorCommand) -> Result<CommandOutput> {
...
-                &cmd.actor_ref, &host
-            ),
+        let text = format!(
+            "Start actor request received: {}, host: {}",
+            &cmd.actor_ref, &host
+        );
+        return Ok(CommandOutput::new(
+            text.clone(),
+            HashMap::from([
+                ("result".into(), JsonValue::String(text)),
```
Comment (connorsmith256) [inline crates/wash-lib/src/cli/start.rs]: Curious if `text.into()` works instead of explicitly creating a `JsonValue::String`?

File: crates/wash-lib/src/cli/stop.rs [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    let event = wait_for_provider_stop_event(
+        &mut receiver,
+        Duration::from_millis(timeout_ms),
+        cmd.host_id.to_string(),
+        cmd.provider_id.to_string(),
+    )
+    .await?;
+
+    match event {
+        FindEventOutcome::Success(_) => Ok(CommandOutput::from_key_and_text(
```
Comment (connorsmith256) [inline crates/wash-lib/src/cli/stop.rs]: I support adding JSON output like you did for the start commands. But maybe those should be done in a separate commit. It feels a bit odd to be in a halfway state

**Review by connorsmith256 — COMMENTED** [review]
LGTM, holding on approval with a couple comments and tests passing

**Comment by vados-cosmonic** [comment]
Just fixed the tests -- will rebase once

File: crates/wash-lib/src/cli/start.rs (hunk 1 above)
Comment (vados-cosmonic) [reply crates/wash-lib/src/cli/start.rs]: Yeah, `into()` does work -- switching to that!

- [x] move all `serde_json::Value` usage to `into()`

File: crates/wash-lib/src/cli/stop.rs (hunk 2 above)
Comment (vados-cosmonic) [reply crates/wash-lib/src/cli/stop.rs]: Getting the output was actually necessary for the test -- I can add the output for the stop commands too here if that will make it a bit more comfortable!

- [x] add improved output for stop comands

**Review by connorsmith256 — APPROVED** [review]
:shipit:
